src/main/resources/feedbackQuestionRankOptionsEditFormTemplate.html

```diff
@@ -26,6 +26,36 @@
             </label>
         </div>
     </div>
+    <div class="col-sm-6">
+        <div class="checkbox" data-toggle="tooltip" data-placement="top" data-container="body" title="Ticking this will ensure respondent ranks at least the mentioned number of options.">
+            <label class="bold-label">
+                <input type="checkbox" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSCHECKBOX}"
+                    id="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSCHECKBOX}-${questionNumber}"
+                    ${isMinOptionsToBeRankedEnabled}>
+                Minimum number of options a respondent must rank
+            </label>
+        </div>
+        <br>
+        <div class="pull-right">
+            <input type="number" class="form-control" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSTOBERANKED}"
+                id="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSTOBERANKED}-${questionNumber}" min="1" value="${minOptionsToBeRanked}"/>
+        </div>
+    </div>
+    <div class="col-sm-6">
+        <div class="checkbox" data-toggle="tooltip" data-placement="top" data-container="body" title="Ticking this will ensure respondent ranks at most the mentioned number of options.">
+            <label class="bold-label">
+                <input type="checkbox" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMAXOPTIONSCHECKBOX}"
+                    id="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMAXOPTIONSCHECKBOX}-${questionNumber}"
+                    ${isMaxOptionsToBeRankedEnabled}>
+                Maximum number of options a respondent can rank
+            </label>
+        </div>
+        <br>
+        <div class="pull-right">
+            <input type="number" class="form-control" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMAXOPTIONSTOBERANKED}"
+                id="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMAXOPTIONSTOBERANKED}-${questionNumber}" min="1" value="${maxOptionsToBeRanked}"/>
```

Comment:
The lines `id...` immediately following `<input...` should be indented 8 spaces. I'm leaving just one comment on this to cover all instances in this file as this may change in the future, since we're still in the process of converting these files to use 2 space indentation.